gloo/docs/gloo_routing/virtual_services/security/apikey_auth/_index.md

```diff
@@ -1,30 +1,52 @@
 ---
-title: ApiKey Auth
+title: API Keys
 weight: 40
 description: How to setup ApiKey authentication. 
 ---
 
-There are a few different ways to use Gloo External Auth. In certain cases – such as protecting routes with
-secure, long-lived UUIDs – it may be desirable to secure a set of routes with **ApiKey Auth**. Keep in mind
-that your routes are only as secure as your apikeys; securing apikeys and proper apikey rotation is
-up to the user, thus the security of the routes is up to the user.
+{{% notice note %}}
+{{< readfile file="static/content/enterprise_only_feature_disclaimer" markdown="true">}}
+{{% /notice %}}
 
-In **ApiKey Auth**, the Gloo VirtualService containing the routes can be configured with a label selector to identify
-multiple valid apikey secrets, or direct references to apikey secrets. When the virtual service configuration changes,
-or when any apikey secret changes, Gloo immediately updates the external auth server with the new configuration. On the
-request path, Envoy asks the external auth service to check the request; any request to a path on that virtual
-service must have a valid apikey in the `api-key` header or will be denied.
+{{% notice note %}}
+The API keys authentication feature was introduced with **Gloo Enterprise**, release 0.18.5. If you are using an earlier version, this tutorial will not work.
+{{% /notice %}}
 
-## Setup
+{{% notice warning %}}
+{{< readfile file="/static/content/extauth_version_info_note" >}}
+{{% /notice %}}
```

Comment:
I think we overuse the notice tags here. Over half the initial screen are warnings/notes and looks kind of off.

I think we can save space here by combining the enterprise only disclaimer and the version disclaimer into one notice (like it was before)
i.e.,
```
{{% notice note %}}
The API keys authentication feature was introduced with **Gloo Enterprise**, release 0.18.5. If you are using an earlier version, this tutorial will not work.
{{% /notice %}}
```

Also, I think we can/should move the notice about the new AuthConfig format/CRD to the place we first use it on a virtual service

Reply:
I was trying to reuse the `static/content/enterprise_only_feature_disclaimer`, but I agree that it is too crowded up there. I'll fix that.